src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+const ChatLog = (props) => {
+  return (
+    <section>
+      {props.entries.map((message) => {
+        return (
+          <ChatEntry 
+            key={message.id}
+            id={message.id}
+            sender={message.sender}
+            body={message.body}
+            timeStamp={message.timeStamp}
+            liked={message.liked}
+            onHandleLikeUpdate={props.onHandleLikeUpdate}
+            onUpdateLikeCount={props.onUpdateLikeCount}
+          />
+    )})}
+    </section>
+  );
+}
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
👍 Nice use of `shape` to describe the data the values within the array should have.